3rdparty/downward/CMakeLists.txt

```diff
@@ -5,18 +5,21 @@ find_package(catkin REQUIRED)
 
 catkin_package()
 
-add_custom_command(
-  OUTPUT installed
-  COMMAND \$\(MAKE\) -f ${PROJECT_SOURCE_DIR}/Makefile
```

Comment:
I'm very sorry, but I still cannot understand.

```diff
-add_custom_command(
-  OUTPUT installed
-  COMMAND \$\(MAKE\) -f ${PROJECT_SOURCE_DIR}/Makefile
```

I didn't add but remove these lines.
What is better solution which I should do?